plugins/services/src/js/components/forms/ContainerServiceFormSection.js

```diff
@@ -456,7 +456,7 @@ class ContainerServiceFormSection extends Component {
             <FieldLabel className="text-no-transform">CPUs</FieldLabel>
             <FieldInput
               min="0.001"
-              name={cpusPath}
+              name="cpus"
```

Comment:
it should stay `name={cpusPath}` otherwise it doesn't allow any changes for pods